clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10541,8 +10582,10 @@ void SPIRVTranslator::ConstructJob(Compilation &C, const JobAction &JA,
     TranslatorArgs.push_back(C.getArgs().MakeArgString(Filename));
   }
 
-  auto Cmd = std::make_unique<Command>(JA, *this, ResponseFileSupport::None(),
-      TCArgs.MakeArgString(getToolChain().GetProgramPath(getShortName())),
+  auto ToolName = UseSPIRVBackend ? "llc" : getShortName();
+  auto Cmd = std::make_unique<Command>(
+      JA, *this, ResponseFileSupport::None(),
```

Comment:
It looks like we are doing this from the `SPIRVTranslator::ConstructJob` function. This is probably the easiest place to make the cahnge, but from an architectural point of view I would have expected us to not create the `SPIRVTranslator` job if this option is passed but instead create a new `Backend` job or something, however I'll leave the final call on this to Mike.

Reply:
I think we would eventually remove SPIR-V Translator Toolchain and use clang toolchain for adding this job. If creating a new backend job is not too complicated, I can attempt it here.

Thanks

Reply:
For now, leaving it here is OK to get the ball rolling.  Please add a TODO to separate out a unique `llc` JobAction to be used so we aren't overloading the SPIRVTranslator job.

Reply:
Another TODO and will be addressed sooner than later. Thanks